### PR TITLE
`z_player_call` OK

### PR DESCRIPTION
**Pull request by AngheloAlf**
Before opening this PR, ensure the following:
- `./format.sh` was run to apply standard formatting.
- `make` successfully builds a matching ROM.
- No new compiler warnings were introduced during the build process.
    - Can be verified locally by running `tools/warnings_count/check_new_warnings.sh`
- New variables & functions should follow standard naming conventions.
- Comments and variables have correct spelling.
---

### CODE REVIEW

File: src/code/z_player_call.c [hunk 1]
```diff
@@ -1,11 +1,60 @@
...
+    ACTOR_PLAYER,
+    ACTORCAT_PLAYER,
+    FLAGS,
+    GAMEPLAY_KEEP,
+    sizeof(Player),
+    (ActorFunc)PlayerCall_Init,
+    (ActorFunc)PlayerCall_Destroy,
+    (ActorFunc)PlayerCall_Update,
+    (ActorFunc)PlayerCall_Draw,
+};
```
Comment (EllipticEllipsis) [inline src/code/z_player_call.c:27]: So this is part of Player? It has Player's InitVars in it...

**Review by engineer124 — APPROVED** [review]
LGTM

File: src/code/z_player_call.c (hunk 1 above)
Comment (AngheloAlf) [reply src/code/z_player_call.c:27]: Well, yes, it is part of the Player system

Comment (EllipticEllipsis) [reply src/code/z_player_call.c:27]: But it has PlayerCall functions in it.

Comment (AngheloAlf) [reply src/code/z_player_call.c:27]: This topic was discussed this morning on the Discord server
https://discord.com/channels/688807550715560050/945521329002672178/945712870694457365

**Review by EllipticEllipsis — APPROVED** [review]

**Review by Kenix3 — APPROVED** [review]

**Pull request merged**